cap-ci/jobs/minibroker-integration-tests.tmpl

```diff
@@ -0,0 +1,57 @@
+{{ define "jobs_minibroker-integration-tests" }}
+- name: minibroker-integration-tests-{{ .scheduler }}-{{ .backend }}-{{ .avail }}
+  serial_groups: [{{ .scheduler }}-{{ .backend }}-{{ .avail }}]
+  public: false
+  plan:
+  - get: catapult
+  - get: minibroker-integration-tests
+  - get: s3.minibroker
+  - get: mits-gh-release
+  - get: {{ .backend }}-pool.kube-hosts
+    {{- if ne .position 0 }}
+    passed:
+    - {{ index .jobs_enabled (sub .position 1) }}-{{ .scheduler }}-{{ .backend }}-{{ .avail }}
+    {{- end }}
+    trigger: true
+  - task: test-{{ .scheduler }}
+    {{- if has .workertags .backend }}
+    tags: [{{ index .workertags .backend }}]
+    {{- end }}
+    timeout: 1h30m
+    input_mapping:
+      pool.kube-hosts: {{ .backend }}-pool.kube-hosts
+    config:
+      platform: linux
+      image_resource:
+        type: registry-image
+        source:
+          repository: splatform/catapult
+      inputs:
+      - name: catapult
+      - name: minibroker-integration-tests
+      - name: pool.kube-hosts
+      - name: s3.minibroker
+      - name: mits-gh-release
+      params:
+        QUIET_OUTPUT: true
+        DOWNLOAD_CATAPULT_DEPS: false
+        ENABLE_EIRINI: {{ eq .scheduler "eirini" }}
+{{- if eq .avail "ha" }}
+        HA: true
+{{- end }}
+{{- if eq .avail "all" }}
+{{- print .allbells }}
+{{- end }}
+        KUBECF_TEST_SUITE: mits
+      run:
+        path: "/bin/bash"
+        args:
+          - -c
+          - |
+            {{ tmpl.Exec "scripts_obtain_kubeconfig" | indent 12 | trimSpace }}
+            {{ tmpl.Exec ( print "scripts_import_" .backend ) | indent 12 | trimSpace }}
+            {{ tmpl.Exec "scripts_test" | indent 12 | trimSpace }}
```

Comment:
Instead of calling this generic `template`, you call the specialized one according to my comment on that file.

Reply:
https://github.com/SUSE/cap/pull/49/commits/7565891d8877174309127420f89b03cc51efc00d